examples/plugins/iowait/main.go

```diff
@@ -0,0 +1,138 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"log"
+	"net"
+	"net/http"
+	"os"
+	"os/exec"
+	"strconv"
+	"strings"
+	"time"
+)
+
+func main() {
+	hostname, _ := os.Hostname()
+	var (
+		addr   = flag.String("addr", "/var/run/scope/plugins/iowait.sock", "unix socket to listen for connections on")
+		hostID = flag.String("hostname", hostname, "hostname of the host running this plugin")
+	)
+	flag.Parse()
+
+	log.Println("Starting...")
+
+	// Check we can get the iowait for the system
+	_, err := iowait()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	os.Remove(*addr)
```

Comment:
I was expecting that most plugins would be not-in-go, so I figured an example and documentation might be more useful than a library, but I'll open an issue for it.